airbyte-webapp/src/views/Connector/ConnectorDocumentationLayout/DocumentationPanel.tsx

```diff
@@ -57,20 +65,49 @@ export const DocumentationPanel: React.FC = () => {
     setDocumentationPanelOpen(false);
   }, [setDocumentationPanelOpen, location.pathname]);
 
+  const [activeTab, setActiveTab] = useState("setupGuide");
+  const tabs = [
+    {
+      id: "setupGuide",
+      name: <FormattedMessage id="sources.documentationPanel.tabs.setupGuide" />,
+    },
+    {
+      id: "schema",
+      name: <FormattedMessage id="sources.documentationPanel.tabs.schema" />,
+    },
+    {
+      id: "erd",
+      name: <FormattedMessage id="sources.documentationPanel.tabs.erd" />,
+    },
+  ];
+
   return isLoading || documentationUrl === "" ? (
     <LoadingPage />
   ) : (
     <div className={styles.container}>
-      <PageHeader withLine title={<FormattedMessage id="connector.setupGuide" />} />
-      <Markdown
-        className={styles.content}
-        content={
-          docs && !error
-            ? prepareMarkdown(docs, isCloudApp() ? "cloud" : "oss")
-            : formatMessage({ id: "connector.setupGuide.notFound" })
-        }
-        rehypePlugins={urlReplacerPlugin}
-      />
+      {selectedConnectorDefinition && isSourceDefinition(selectedConnectorDefinition) && showRequestSchemaButton && (
```

Comment:
Tabs will only be shown if the experiment flag is set to true, preventing showing up in OSS